src/dev/build/tasks/replace_favicon.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { copy, Task } from '../lib';
+
+export const ReplaceFavicon: Task = {
+  description: 'Replacing favicons with built version',
+
+  async run(config, log, build) {
+    await copy(
+      config.resolveFromRepo('src/core/server/core_app/assets/favicons/favicon.built.png'),
```

Comment:
nit: could we use `distribution` instead of `built` in the filename? `built` seems to be a misnomer.

Reply:
Yep, will do!